admin/tests/unit/controllers/export-dhis.spec.js

```diff
@@ -0,0 +1,94 @@
+describe('dhis2 export controller', () => {
```

Comment:
are there e2e tests for the admin console? i'm not seeing any, so i made a unit test for the controller

Reply:
actually, i'm going to need advice on how you'd pursue this. the integration test did not shape up well and also seems low value since you can't connect it to the api result

Reply:
If admin e2e tests don't Just Work™ (ie you can't just navigate the URL to there and click away) then probably raise a ticket about adding e2e support to the admin app and we can rely on manual testing for this one.